index.html

```diff
@@ -0,0 +1,227 @@
+    <!-- About Section -->
+    <section class="main_containers">
+        <div class="about-container">
+            <div class="minicontainers about1">
+                <h1 class="about-primary-text">About Me</h1>
+                <p class="about-secondary-text">Hello I’m a software developer! I can help you 
+                    build a product , feature or website. 
+                    Look through some of my work and experience! 
+                    If you like what you see and have a project 
+                    you need coded, don’t hesitate to contact me.
+                </p>
+                <div class="social_media">
+                    <h2 class="tertiary-text">Let's connect</h2>
+                    <ul class="socialmedia-icons">
+                        <li><a href="#"><img src="./assets/images/github.svg" alt="github"></a> </li>
+                        <li><a href="#"><img src="./assets/images/Linkedin icon.svg" alt="Linkedin"></a> </li>
+                        <li><a href="#"><img src="./assets/images/angel list.svg" alt="angellist"></a> </li>
+                        <li><a href="#"><img src="./assets/images/medium.svg" alt="Medium"></a> </li>
+                    </ul>
+                  </div>
+                  <button class="btn">Get my resume</button>
+            </div>
+            <div class="minicontainers about2">
+                <ul class="skills">
+                    <li class="skills-title">
+                        <h3 class="skills-primary-text">Languages</h3>
+                        <img class="dropdown" src="./assets/images/dropdown-bottom.svg" alt="">
+                    </li>
+                    <li>
+                        <ul class="language-components">
+                            <li>
+                                <span><img class="logos javascriptlogo" src="./assets/images/javascriptimage.svg" alt=""></span>
+                                <p>Javascript</p>
+                            </li>
+                            <li>
+                                <span><img class="logos htmllogo" src="./assets/images/Htmlimg.svg" alt=""></span>
+                                <p>Html</p>
+                            </li>
+                            <li>
+                                <span><img class="logos csslogo" src="./assets/images/cssimg.svg" alt=""></span>
+                                <p>Css</p>
+                            </li>
+                        </ul>
+                    </li>
+                    <li class="skills-title-border">
+                        <h3 class="skills-primary-text">Frameworks</h3>
+                        <span><img src="./assets/images/dropdownleft.svg" alt=""></span>
+                    </li>
+                    <li>
+                        <h3 class="skills-primary-text">Skills</h3>
+                        <span><img src="./assets/images/dropdownleft.svg" alt=""></span>
+                    </li>
+                </ul>
+            </div>
```

Comment:
[ ] Please note that your design is required to match the Figma design.
- The `languages` does not resemble the Figma design 
- The `frameworks and skills do not resemble the Figma design 
- There is a border in your application that is not present in the Figma design (when other students click on a section on Figma, there is a border created, please be careful and make sure any border is added is in the Figma design)

###
>This is your design
![Screenshot from 2023-06-03 17-11-44](https://github.com/Bettenoch/My_Portfolio/assets/69245974/2251a828-30fa-49a8-b6db-8641de0821eb)

###
> This is the Figma design
![Screenshot from 2023-06-03 17-12-20](https://github.com/Bettenoch/My_Portfolio/assets/69245974/e75a933d-b571-4615-8b2e-57baf73e9e42)